source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -437,10 +439,12 @@ export class NumberPropertyEditor extends SingleInputPropertyEditor {
 
     protected createInput(context: PropertySheetContext): Adaptive.Input {
         let input = new Adaptive.NumberInput();
-        input.defaultValue = this.getPropertyValue(context);
+
+        input.defaultValue = this.getPropertyValue(context) ? this.getPropertyValue(context) : this.defaultValue;
```

Comment:
Avoids calling `getPropertyValue()` twice :)

```suggestion
        input.defaultValue = this.getPropertyValue(context) ?? this.defaultValue;
```